valinta-tulos-service/src/main/scala/fi/vm/sade/valintatulosservice/ValintatulosService.scala

```diff
@@ -140,51 +153,62 @@ class ValintatulosService(vastaanotettavuusService: VastaanotettavuusService,
     timed("Fetch hakemusten tulos for haku: " + hakuOid, 1000) (
       for {
         haku <- hakuService.getHaku(hakuOid).right.toOption
+        hakukohdeOids <- hakuService.getHakukohdeOids(hakuOid).right.toOption
```

Comment:
Tässä virhekäsittely muuttuu hieman. Aiemmin jos koulutuksen alkamiskautta ei ole saatu haettua, on palautettu `Right(Iterator.empty)`. Nyt tulee `Left(jokupoikkeus)` josta varmaankin rajapinta palauttaa 500 tms. Sinäänsä parempi näin, jos vaan uskaltaa rikkoa mahdollisten kutsujien virheenkäsittelyn :)